randovania/games/planets_zebeth/exporter/game_exporter.py

```diff
@@ -42,8 +50,64 @@ def export_params_type(self) -> type[GameExportParams]:
     def _do_export_game(
         self,
         patch_data: dict,
-        export_params: GameExportParams,
+        export_params: PlanetsZebethGameExportParams,
         progress_update: status_update_lib.ProgressUpdateCallable,
     ) -> None:
-        assert isinstance(export_params, PlanetsZebethGameExportParams)
-        raise RuntimeError("Needs to be implemented")
+        # Check if dotnet is available
+        dotnet_ran_fine = False
+        try:
+            dotnet_process = subprocess.run(["dotnet", "--info"], check=False)
+            dotnet_ran_fine = dotnet_process.returncode == 0
+        except FileNotFoundError:
+            dotnet_ran_fine = False
+
+        monitoring.set_tag("planets_dotnet_ran_fine", dotnet_ran_fine)
+
+        if not dotnet_ran_fine:
+            raise UnableToExportError(
+                "You do not have .NET installed!\n"
+                "Please ensure that it is installed and located in PATH. It can be installed "
+                "from here:\n"
+                "https://aka.ms/dotnet/download"
+            )
```

Comment:
This feels like something that should be extracted to somewhere else so that am2r and planets aren't copypasting the code here.